jobboard/models.py

```diff
@@ -1,38 +1,42 @@
 from django.db import models
-from recruiter.models import Company
+from recruiter.models import Company, Recruiter
 import datetime
 
 
 class Region(models.Model):
     name = models.CharField(max_length=255)
 
-    def __str__(self):
```

Comment:
why do you change this?

Reply:
It was auto changed for some reason. I'll change it back to what it was